fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -80,21 +95,37 @@ fun TabsTray(
                         if (displayTabsInGrid) {
                             TabGrid(
                                 tabs = normalTabs,
+                                onTabClose = interactor::onTabClosed,
+                                onTabMediaClick = interactor::onMediaClicked,
+                                onTabClick = onTabClick,
+                                onTabLongClick = interactor::onTabLongClicked,
                             )
                         } else {
                             TabList(
                                 tabs = normalTabs,
+                                onTabClose = interactor::onTabClosed,
+                                onTabMediaClick = interactor::onMediaClicked,
+                                onTabClick = onTabClick,
+                                onTabLongClick = interactor::onTabLongClicked,
                             )
                         }
                     }
                     Page.PrivateTabs -> {
                         if (displayTabsInGrid) {
                             TabGrid(
                                 tabs = privateTabs,
+                                onTabClose = interactor::onTabClosed,
+                                onTabMediaClick = interactor::onMediaClicked,
+                                onTabClick = onTabClick,
+                                onTabLongClick = interactor::onTabLongClicked,
                             )
                         } else {
                             TabList(
                                 tabs = privateTabs,
+                                onTabClose = interactor::onTabClosed,
+                                onTabMediaClick = interactor::onMediaClicked,
+                                onTabClick = onTabClick,
+                                onTabLongClick = interactor::onTabLongClicked,
```

Comment:
My impression/intention is to revisit and possibly refactor this **after** the Inactive Tabs have been added in, as that ticket may directly impact how the code is shared between the Normal & Private tab pages.